tests/test_cli.py

```diff
@@ -108,7 +141,6 @@ def store(data, section):
     monkeypatch.setattr('stups_cli.config.store_config', store)
```

Comment:
I believe because it integrates nicely with `pytest` and [it was created before](https://holgerkrekel.net/2009/03/03/monkeypatching-in-unit-tests-done-right/) (2009) `mock` package got integrated in the builtin Python 3.3+ (2012).